### DISCUSSION TITLE
Grid profile addition request

### DISCUSSION THREAD

**Discussion by RonnieLast** [tbnobody/OpenDTU#1678] [General]
Hi, how do I request a new grid profile addition? The Irish grid spec is slightly different to the European standard of EN50549-1:2019. Is that something which can be added into openDTU or does it have to happen on Hoymiles' software first and then get added into OpenDTU? 

If it can be added into OpenDTU can I add a row into the "Grid Profile Parser" page, linking to the differences between the EN50549-1:2019 spec and the Irish IS EN50549-1:2019 spec?
https://github.com/tbnobody/OpenDTU/wiki/Grid-Profile-Parser

**Comment by stefan123t** [comment]
@RonnieLast I have documented most of the known Grid Profiles in the Wiki: 
https://github.com/lumapu/ahoy/wiki/Protocol#file-name-nf_en_50549-12019

Can you double check which details / values would be different to the other already known Grid Profiles, eg. NF_EN_50549-1:2019 ?

I have found the following Certificate on the Hoymiles pages 

**Certificate of compliance**  
https://www.hoymiles.com/wp-content/uploads/2022/09/HM-Series_EN-50549-1-2019_EU_Certificate_V20210201.pdf

and a research presentation on the various national implementations of the EN-50549-1:2019 standards in Europe

**Grid Codes in Europe - Overview on the current requirements in European codes and national interconnection standards**
https://www.researchgate.net/publication/338800967_Grid_Codes_in_Europe_-_Overview_on_the_current_requirements_in_European_codes_and_national_interconnection_standards

Yes, we have to appreciate the *s* in standard**s** :grinning: 

**Ireland**

**Applicable to ABCD**

Rate Of Change Of Frequency (ROCOF) withstand capability: 1 Hz/s (500 ms) 

Limited Frequency Sensitive Mode – Over frequency (LFSM-O) [Art. 13-2 (a-g)]
Requirement Droop: 4%
Threshold: 50.2 Hz
Delay: none

**Applicable to Type B**

Reactive power capability for Type B PPM
Reactive power range Q/Pmax: -0.33 to 0.33
Voltage range: Not specified

FRT capability of Type B PPMs
Voltage parameters
Uret: 0.15
Uclear: 0.15
Urec1: 0.15
Urec2: 0.85
Time parameters
tclear: 0.25
trec1: 0.25
trec2: 0.25
trec3: 2.9

Fast fault current injection capability of Type B PPMs
FFCI: Yes, no parameters specified

**Reply by RonnieLast** [reply]
Thank you for your comprehensive response Stefan.

The delta between the I.S. (Irish Standards) and EN50549-1:2019 are limited to the following:

- Over voltage, Stage 1: 269V / 468V with a 70s clearance time. 
- Over voltage, Stage 2: 281V / 488V with a 0.7s clearance time. 
- Under voltage: 191V / 332V with a 0.7s clearance time. 
- Over frequency: 52Hz with a 0.5s clearance time (& 52.1Hz where a step setting of 0.1Hz is available).
- Under frequency: 47Hz with a 0.5s clearance time (& 46.9Hz where a step setting of 0.1Hz is available).
- ROCOF: 1.0 Hz/s with a 0.6s clearance time. 
- Vector Shift: Is not permitted. 

These are communicated in ESBn (ESB Networks Ireland) document "Conditions Governing the Connection and Operation of Micro-
Generation" dated 03/12/2021, section 2.2 "Interface Protection Settings":

https://www.esbnetworks.ie/docs/default-source/publications/conditions-governing-the-connection-and-operation-of-micro-generation-policy.pdf?sfvrsn=fdcb2515_17

Is this sufficient clarity for you needs?

**Comment by RonnieLast** [comment]
Hi all, can I just bump this as a polite reminder? Can I provide further details to help get this completed?

**Comment by morning-call** [comment]
Just bumping this to see if anything further update to the additional grid profile?

Thanks
